source_sdk/cvar/convar/convar.h

```diff
@@ -0,0 +1,8 @@
+#include <stdbool.h>
+#include <stdint.h>
+
+typedef void * convar;
```

Comment:
Interesting. I like using `void *` for simplicity and learning purposes but maybe I'll start using this throughout the project for clarity.

Reply:
Maybe there could also be type definitions for entities? I'd like to respect what you want in your code when I'm publishing to your repository, but `void *` can be a little abstract to what its pointing to.